### DISCUSSION TITLE
Placeholder for Claimblocks

### DISCUSSION THREAD

**Discussion by NilanFuncke** [GriefPrevention/GriefPrevention#2256] [Q&A]
Hi,

Amazing plugin! But I've got a scoreboard and I want it to show the claimblocks a player has, what's the placeholder for that? I don't mind using plugins like PlaceholderAPI!

**Comment by Jikoo** [comment]
There is a PAPI hook, but it is not provided by GP, and its author does not follow GP issues afaik. https://github.com/pixar02/Expansions
